playbulb-candle/app.js

```diff
@@ -0,0 +1,111 @@
+var r = g = b = 255; // White by default.
+
+document.querySelector('#connect').addEventListener('click', function() {
+  document.querySelector('#state').classList.add('connecting');
```

Comment:
With bluetooth flakiness I happened to find useful to be able to click multiple times the "Connect" button.